e2e/stack_test.go

```diff
@@ -155,20 +156,20 @@ func TestE2E(t *testing.T) {
 	)
 	require.NoError(t, err, "deposit tx")
 
-	client, err := bftclient.New(monomerCometURL.String(), monomerCometURL.String())
+	appchainClient, err := bftclient.New(monomerCometURL.String(), monomerCometURL.String())
```

Comment:
purpose for the rename? I feel I'm missing the intent here.

Reply:
There are a few clients in this file to different processes (ethRPCClient, appchainClient, monomerClient). Disambiguating.